CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.1.0 (TBD, 2021)
+* Enhancements
+   * Converted persistent history files from pickle to compressed JSON
```

Comment:
What will happen if someone with an old pickle-based history file upgrades to 2.1.0?

Reply:
It will print an error and start history from scratch. Since 2.0.0 just came out a few days ago, I doubt this will affect too many people.

Reply:
I updated the error message to include:

> The history file will be recreated when this application exits.